### ISSUE TITLE
Alias to another object

### ISSUE THREAD

**Issue by dy** [jkroso/alias-property#1]
Now that seems to be a really nice for use-case of synonimical properties.
I wonder if you'd consider extending the API to bind to another object's properties.
That'd be super-useful for cases of multiple class instances with shared variables (personally I use that in [gl-waveform](https://ghub.io/gl-waveform)).

Eg.
```js
let a = {total: 1}, b = {}
alias(a, 'total', 'length') // a.length === a.total
alias(a, 'total', b, 'length') // a.total === b.length
alias(a, 'total', b) // a.total === b.total
alias(a, b) // a.* === b.*
```

Or should that better be a separate package @jkroso ?

**Comment by jkroso** [comment]
So you're trying to simulate multiple inheritance?

**Comment by dy** [comment]
@jkroso not exactly. Just mirroring properties from another object. In fact, there is a [mirrorprop](https://ghub.io/mirrorprop) package for that, not so elaborate though.

**Comment by jkroso** [comment]
Yeah alright. Could be cool to `alias(a, a.prototype)`
